chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -96,8 +104,38 @@
           <device>/dev/<%= @boot_device %></device>
         <% end %>
         <initialize config:type="boolean">true</initialize>
-        <partitions config:type="list"/>
         <type config:type="symbol">CT_DISK</type>
+        <disklabel>gpt</disklabel>
+        <partitions config:type="list">
+          <partition>
+            <create config:type="boolean">true</create>
+            <format config:type="boolean">true</format>
+            <fstopt>acl,user_xattr</fstopt>
```

Comment:
Since we are creating a boot partition manually which is not the defaults, I think its a good idea to explicitly mention fstopt. Without this, grub was not getting installed which was due to permissions issue (most likely reason).

For the other partitions, not specific specific fstopt would make yast decide these attrs. and would be set to defaults which work properly. We could set them if we have some specific requirements. If not, keeping them to be decided by yast (not setting them) should be preferred since in future if some defaults change, we need not worry about custom flags creating regressions :).